docs/lib/analytics/fragments/ios/identifyuser.md

```diff
@@ -2,33 +2,38 @@ This call sends information that you have specified about the user to Amazon Pin
 
 ```swift
 func identifyUser() {
-    let userState = AWSMobileClient.default().currentUserState
-    let getIdentityIdTask = AWSMobileClient.default().getIdentityId()
-    getIdentityIdTask.continueWith { (task) -> Any? in
-        if let error = task.error {
-            print("Failed to get identityId: \(error)")
-        }
 
-        guard let identityId = task.result else {
-            print("Missing identityId")
-            return nil
+    var identityId: String?
+    
+    // Get identityId
+    _ = Amplify.Auth.fetchAuthSession(listener: { event in
+        switch event {
+        case .success(let authSession):
+            let cognitoAuthSession = (authSession as? AuthCognitoIdentityProvider)?.getIdentityId()
+            switch cognitoAuthSession {
+            case .success(let identityId):
+                identityId = identityId
+            case .failure(let error):
+                print("Failed to get identity id: \(error)")
+            }
+        case .failure(let error):
+            print("Failed to fetch auth session: \(error)")
         }
+    })
+
+    let location = AnalyticsUserProfile.Location(latitude: 47.606209,
+                                                    longitude: -122.332069,
+                                                    postalCode: "98122",
+                                                    city: "Seattle",
+                                                    region: "WA",
+                                                    country: "USA")
+
+    let userProfile = AnalyticsUserProfile(name: "name",
+                                            email: "name@example.com",
+                                            phoneNumber: "206-266-1000"
+                                            location: location)
 
-        print("Got identityId: \(identityId). UserState: \(userState)")
-        let location = AnalyticsUserProfile.Location(latitude: 47.606209,
-                                                     longitude: -122.332069,
-                                                     postalCode: "98122",
-                                                     city: "Seattle",
-                                                     region: "WA",
-                                                     country: "USA")
-        let properties = ["userState": "\(userState)"]
-        let userProfile = AnalyticsUserProfile(name: "name",
-                                               email: "name@example.com",
-                                               location: location,
-                                               properties: properties)
-        Amplify.Analytics.identifyUser(identityId as String, withProfile: userProfile)
+    Amplify.Analytics.identifyUser(identityId as String, withProfile: userProfile)
```

Comment:
please test the code

Reply:
Ya, the getCurrentUser() and userId work here, it's the signed in or not issue I mention on above. What do you think about it?

Reply:
can remove `as String` after changing to `user: user.userId`

Reply:
Removed